app/endpoints/user_routes.py

```diff
@@ -1,17 +1,26 @@
-from fastapi import FastAPI, HTTPException, APIRouter
-
+from fastapi import FastAPI, HTTPException, APIRouter, status, Depends
 from app.models.user import User
+from app.models.hashing import Hash
+from app.models.oauth import get_current_user
+from fastapi.middleware.cors import CORSMiddleware
+from app.services.connect import user_collection
+from app.services.database import register_user, login_user
+from fastapi.security import OAuth2PasswordRequestForm
 
-router = APIRouter()
 
-from app.services.database import create_user
 
 router = APIRouter()
 
 
-@router.post('/api/user/register', response_model=User)
-async def post_user(user: User):
-    response = await create_user(user.dict())
-    if response:
-        return response
-    raise HTTPException(400, "Something went wrong")
\ No newline at end of file
+@router.post('/api/user/register')
+async def create_user(request:User):
+    if await register_user(request):
+        return {"res":"created"}
+    return {"res":"ID not created"}
```

Comment:
For using the status code and error message on Frontend, return object need to be fixed like the below.
{  
  "message":error-message,  
  "status-code":error-code  
}